### PR TITLE
e-commerce »» ecommerce

### PR DESCRIPTION
**Pull request by pfd**
Fix style for ecommerce

### CODE REVIEW

**Review by ameypv-rudder — APPROVED** [review]

File: docs/destinations/streaming-destinations/adobe-analytics/adobe-analytics-web-device-mode.mdx [hunk 1]
```diff
@@ -49,17 +49,17 @@ According to the mapping done in RudderStack, the events can be sent as particul
...
-2. E-Commerce track events
+2. Ecommerce track events
 3. Video type \(Heartbeat\) track events
 
 <div class="infoBlock">
 For the regular and video type (Heartbeat) <code class="inline-code">track</code> events, it is necessary to map the events in the RudderStack dashboard.
 </div>
 
-For the E-Commerce track events, if the events fall under the following mapping they will be sent accordingly. Otherwise, the mapping should be done in the dashboard.
+For the Ecommerce track events, if the events fall under the following mapping they will be sent accordingly. Otherwise, the mapping should be done in the dashboard.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/adobe-analytics/adobe-analytics-web-device-mode.mdx:59]: Quick question - are we using `Ecommerce` throughout, including the terms that appear in the middle of a sentence?

File: docs/event-spec/ecommerce-events-spec/ordering.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Ordering"
-description: Understand how ecommmerce product order events work in RudderStack.
+description: Understand how ecommerce product order events work in RudderStack.
```
Comment (ameypv-rudder) [inline docs/event-spec/ecommerce-events-spec/ordering.mdx:3]: following up on the above query - we'll need to tweak this accordingly for consistency purposes.

File: docs/destinations/streaming-destinations/adobe-analytics/adobe-analytics-web-device-mode.mdx (hunk 1 above)
Comment (ishachopra0401) [reply docs/destinations/streaming-destinations/adobe-analytics/adobe-analytics-web-device-mode.mdx:59]: In the favor of using ecommerce in between

File: docs/event-spec/ecommerce-events-spec/ordering.mdx (hunk 2 above)
Comment (pfd) [reply docs/event-spec/ecommerce-events-spec/ordering.mdx:3]: this one had 3 ms 😱

Comment (ameypv-rudder) [reply docs/event-spec/ecommerce-events-spec/ordering.mdx:3]: seems dubious 🤔

**Pull request merged**